backend/amd64/proc.ml

```diff
@@ -60,12 +60,12 @@ let win64 = Arch.win64
    Under Unix:
      rdi, rsi, rdx, rcx, r8, r9: C function arguments
      xmm0 - xmm7: C function arguments
-     rbx, rbp, r12-r15 are preserved by C
+     r12-r15 are preserved by C
```

Comment:
Where is this diff coming from? Why are `rbx` and `rbp` no longer preserved by C?

Reply:
Hmm I looked at this on ocaml-jst, but I can't find where my diff has gone to.  Will update

Reply:
This is what's needed: https://github.com/ocaml-flambda/ocaml-jst/pull/243

Reply:
Thanks! What is the plan for the diff between BACKPORT comment and the corresponding code in ocaml-jst#243 that doesn't use rbx and rbp?